src/chapters/orm/studio.rst

```diff
@@ -0,0 +1,114 @@
+
+Getting Ready
+-------------
+
+In your ``coding-events`` repository, checkout out the `add-persistent-category branch <https://github.com/LaunchCodeEducation/coding-events/tree/add-persistent-category>`_ and create a new branch, ``abstract-entity-studio``, off of that branch.
+
+.. admonition:: Note
+
+   You can also branch off of the branch you created in the exercises, if you like. We have provided the branch mentioned to make sure you have good starter code for the studio, even if you didn't complete the exercises. 
+
+Your Tasks
+----------
+
+Let's look at both the ``Event`` and ``EventCategory`` classes. There are four class members that are (essentially) the same in both classes:
+
+- The ``id`` field
+- The ``getId`` method
+- The ``equals`` method
+- The ``hashCode`` method
+
+.. admonition:: Note
+
+   If you have a super-sharp eye, you'll notice that the two ``equals`` methods aren't *exactly* the same. For example, the next-to-last line of this method in ``Event`` looks like this:
+
+   .. sourcecode:: java
+
+      Event Event = (Event) o;
+
+   while in ``EventCategory`` this line is slightly different;
+
+   .. sourcecode:: java
+
+      EventCategory that = (EventCategory) o;
+
+   This minor difference will not matter in the long run, as we'll see later.
+
+Your objective in this studio is to reduce this repetition by creating a class that contains the duplicated/shared code, which both ``Event`` and ``EventCategory`` can inherit.
+
+Create and Extend ``AbstractEntity``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the ``models`` package, create a new class named ``AbstractEntity``. Add the ``abstract`` keyword to the class declaration. We do this because, while we want to share code across our two models, we'll never need to create an instance of ``AbstractEntity``. If you need a quick refresher, review our section on :ref:`abstract-classes`.
+
+.. admonition:: Tip
+
+   This class name seems a little odd at first, but it makes sense after a brief explanation. 
+   
+   It is a commonly-used convention in Java to prefix the name of an abstract class with ``Abstract``, to make it obvious to other developers that they can't create instances of the class. 
+
+   Additionally, *every* entity class that we create will extend ``AbstractEntity``. (Recall that an entity is a class/object that can be stored in a database.)
+
+Finally, in the class declarations of both ``Event`` and ``EventCategory``, extend ``AbstractEntity``.
+
+Move Duplicated Code Into the Superclass
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open up the ``Event`` class file. Move the 4 class members that we identified as being duplicated across both models up into ``AbstractEntity``. Now your ``Event`` class will *inherit* these members rather than define them itself.
```

Comment:
Word removal suggestion, "Move the four duplicate class members that we identified into ``AbstractEntity``.